### PR TITLE
Fixed adding subspace not working

### PR DESCRIPTION
**Pull request by Daboo1998**
Adding subspace sent an empty post request, I have fixed that

Nifty: https://momentum.nifty.pm/l/VC3juVKeTo subtask: Admin panel: create subspace error

### CODE REVIEW

File: src/core/models/Space/Space.ts [hunk 1]
```diff
@@ -171,7 +171,7 @@ const Space = types
       };
 
       yield self.addSubSpaceRequest.send(api.spaceRepository.create, {
-        newSpace
+        space: newSpace
```
Comment (dmitry-yudakov) [inline src/core/models/Space/Space.ts:174]: It's pity typescript isn't catching these for us...

File: src/scenes/spaceAdmin/pages/SpaceAdminPage/components/templates/SubSpacesPanel/components/AddSubSpaceDialog/AddSubSpaceDialog.tsx [hunk 2]
```diff
@@ -36,7 +36,7 @@ const AddSubSpaceDialog: FC<PropsInterface> = (props) => {
         <ToastContent
           headerIconName="alert"
           title={t('titles.alert')}
-          text={t('spaceAdmin.subSpaces.addSubSpaceDialog.success')}
+          text={t('messages.subSpaceCreateSuccess')}
```
Comment (dmitry-yudakov) [inline src/scenes/spaceAdmin/pages/SpaceAdminPage/components/templates/SubSpacesPanel/components/AddSubSpaceDialog/AddSubSpaceDialog.tsx:39]: and these too... :)

File: src/core/models/Space/Space.ts (hunk 1 above)
Comment (Daboo1998) [reply src/core/models/Space/Space.ts:174]: Yep :(

File: src/scenes/spaceAdmin/pages/SpaceAdminPage/components/templates/SubSpacesPanel/components/AddSubSpaceDialog/AddSubSpaceDialog.tsx (hunk 2 above)
Comment (Daboo1998) [reply src/scenes/spaceAdmin/pages/SpaceAdminPage/components/templates/SubSpacesPanel/components/AddSubSpaceDialog/AddSubSpaceDialog.tsx:39]: Indeed, but those are harder to catch for TS

File: src/core/models/Space/Space.ts (hunk 1 above)
Comment (dmitry-yudakov) [reply src/core/models/Space/Space.ts:174]: Good task for some future Friday Hack :) 

I spent about an hour once trying to fix it but then my conscience made me drop it and go work on actual task :))

File: src/scenes/spaceAdmin/pages/SpaceAdminPage/components/templates/SubSpacesPanel/components/AddSubSpaceDialog/AddSubSpaceDialog.tsx (hunk 2 above)
Comment (dmitry-yudakov) [reply src/scenes/spaceAdmin/pages/SpaceAdminPage/components/templates/SubSpacesPanel/components/AddSubSpaceDialog/AddSubSpaceDialog.tsx:39]: We don't know how yet. The whole translation is just an object somewhere so it should be doable. Another good Friday Hack or dev party task :))

**Review by dmitry-yudakov — APPROVED** [review]

**Review by aminbksi — APPROVED** [review]

**Pull request merged**